daal4py/sklearn/utils/test/test_utils.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from daal4py.sklearn import _utils
+
+
+def injected_get_daal_version():
+    return (2020, "C", 100)
+
+
+@pytest.mark.parametrize(
+    "required_version,expected",
+    [
+        ((2020, "C", 100), True),
+        ((2019, "C", 100), True),
+        ((2020, "B", 100), True),
+        ((2020, "C", 99), True),
+        ((2021, "C", 100), False),
+        ((2020, "D", 100), False),
+        ((2020, "C", 101), False),
+        ((2019, "B", 99), True),
+        ((2021, "D", 101), False),
+        (((2020, "C", 100), (2021, "D", 101)), True),
+        (((2019, "C", 100), (2018, "C", 100)), True),
+        (((2019, "C", 100), (2018, "C", 100)), True),
+        (((2021, "C", 100), (2022, "C", 100)), False),
```

Comment:
This is helpful and nice to have. In terms of scope of tuples I think realistic values that could be worth having checks for with the 3rd number are 0, 1, 2, 100, 101, 102, 200, 201, 202 (where minor and patch are the hundreds and ones digits). Not that all of these have to be tested but maybe worth having a 1 instead of a 99, and swapping one of these hundreds for something with minor version 0 (single digit). Also I am not sure what the letter is but I've only ever seen "P"